pkg/kapp/diff/change_set_with_versioned_rs.go

```diff
@@ -16,6 +17,9 @@ const (
 	versionedResAnnKey        = "kapp.k14s.io/versioned"               // Value is ignored
 	versionedResOrigAnnKey    = "kapp.k14s.io/versioned-keep-original" // Value is ignored
 	versionedResNumVersAnnKey = "kapp.k14s.io/num-versions"
+	maxDurationAnnKey         = "kapp.k14s.io/max-duration"
+	lastRenewTimeAnnKey       = "kapp.k14s.io/last-renewed-time"
+	updateStrategyAnnKey      = "kapp.k14s.io/update-strategy"
```

Comment:
Remove this as it is not getting used now, golint is also failing due to this only.